nodes/scheduling/nodes-scheduler-about.adoc

```diff
@@ -88,3 +83,11 @@ xref:../../nodes/scheduling/nodes-scheduler-pod-affinity.adoc#nodes-scheduler-po
 
 These advanced scheduling features allow administrators
 to specify which node a pod can be scheduled on and to force or reject scheduling relative to other pods.
+
```

Comment:
I like the idea of having this as a separate topic. But I defer. 
Should this come before _Scheduler use cases_? First the default scheduler, then the advanced?

Reply:
Good suggestion, makes sense to me, I'll move it!